webapp/src/main/webapp/themes/tenderfoot/templates/body/individual/individual--foaf-person.ftl

```diff
@@ -181,19 +181,17 @@ ${stylesheets.add(
 	'<link rel="stylesheet" href="${urls.base}/css/individual/individual.css" />',
 	'<link rel="stylesheet" href="${urls.base}/css/individual/individual-vivo.css" />',
 	'<link rel="stylesheet" href="${urls.base}/js/jquery-ui/css/smoothness/jquery-ui-1.12.1.css" />',
-	'<link rel="stylesheet" type="text/css" href="${urls.base}/css/jquery_plugins/qtip/jquery.qtip.min.css" />'
```

Comment:
The comma before needs removed as well or the freemarker template complains.

Reply:
Thanks for catching that! I've fixed it up, so please take a look when you get a chance. Appreciate it!